tests/integration-tests/pom.xml

```diff
@@ -440,6 +440,18 @@
          <artifactId>mockito-core</artifactId>
          <scope>test</scope>
       </dependency>
+
+      <dependency>
+         <groupId>org.bouncycastle</groupId>
+         <artifactId>bcprov-jdk18on</artifactId>
+         <scope>test</scope>
+      </dependency>
+
+      <dependency>
+         <groupId>org.bouncycastle</groupId>
+         <artifactId>bcpkix-jdk18on</artifactId>
+         <scope>test</scope>
+      </dependency>
```

Comment:
Doesnt seem like these should be needed, arent they brought in by the broker?

Reply:
good catch, thanks!